### PR TITLE
Update note section

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9235

### CODE REVIEW

File: windows/client-management/mdm/networkqospolicy-csp.md [hunk 1]
```diff
@@ -25,7 +25,11 @@ The following actions are supported:
 - Layer 3 tagging using a differentiated services code point (DSCP) value
 
 > [!NOTE]
-> The NetworkQoSPolicy configuration service provider is officially supported for devices that are Intune managed and Azure AD joined. Currently, this CSP is not supported on Azure AD Hybrid joined devices and for devices using GPO and CSP at the same time. The minimum operating system requirement for this CSP is Windows 10, version 2004. This CSP is supported only in Microsoft Surface Hub prior to Window 10, version 2004.
+> The NetworkQoSPolicy configuration service provider is officially supported for devices that are Intune managed and Azure AD joined. Currently, this CSP is not supported on following devices:
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/networkqospolicy-csp.md:28]:
```suggestion
> The NetworkQoSPolicy configuration service provider is officially supported for devices that are Intune managed and Azure AD joined. Currently, this CSP is not supported on the following devices:
```

File: windows/client-management/mdm/networkqospolicy-csp.md [hunk 2]
```diff
@@ -25,7 +25,11 @@ The following actions are supported:
...
+> - Azure AD Hybrid joined devices
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/networkqospolicy-csp.md:29]:
```suggestion
> - Azure AD Hybrid joined devices.
```

File: windows/client-management/mdm/networkqospolicy-csp.md [hunk 3]
```diff
@@ -25,7 +25,11 @@ The following actions are supported:
...
+> - Devices, that use both GPO and CSP at the same time
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/networkqospolicy-csp.md:30]:
```suggestion
> - Devices that use both GPO and CSP at the same time.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**